Raven.Database/Server/WebApi/RequestManager.cs

```diff
@@ -269,7 +271,57 @@ public void Dispose()
 			return response;
 		}
 
-		/// <summary>
+	    private long PreActionLog(IResourceApiController controller)
+	    {
+            LogHttpRequestStatsParams logHttpRequestStatsParam = null;
```

Comment:
You should only do all this work here if and only if you have log.IsDebugEnabled